platform-emf/src/main/java/com/softicar/platform/emf/attribute/field/foreign/entity/input/browse/EmfEntityInputBrowseColumnHandler.java

```diff
@@ -0,0 +1,44 @@
+package com.softicar.platform.emf.attribute.field.foreign.entity.input.browse;
+
+import com.softicar.platform.dom.elements.DomDiv;
+import com.softicar.platform.dom.event.IDomClickEventHandler;
+import com.softicar.platform.dom.event.IDomEvent;
+import com.softicar.platform.emf.EmfCssClasses;
+import com.softicar.platform.emf.data.table.IEmfDataTableCell;
+import com.softicar.platform.emf.data.table.column.handler.EmfDataTableRowBasedColumnHandler;
+import com.softicar.platform.emf.entity.IEmfEntity;
+import java.util.function.Consumer;
+
+class EmfEntityInputBrowseColumnHandler<E extends IEmfEntity<E, ?>> extends EmfDataTableRowBasedColumnHandler<E, String> {
+
+	private final Consumer<E> clickConsumer;
+
+	public EmfEntityInputBrowseColumnHandler(Consumer<E> clickConsumer) {
+
+		this.clickConsumer = clickConsumer;
+	}
+
+	@Override
+	public void buildCell(IEmfDataTableCell<E, String> cell, E entity) {
+
+		cell.appendChild(new NameDiv(entity));
+	}
+
+	private class NameDiv extends DomDiv implements IDomClickEventHandler {
```

Comment:
There is a UX problem with this element: The user needs to aim for the text, quite precisely. Clicking the cell/row does not suffice. There are even dead zones between entries. Those dead zones should not exist.
![image](https://user-images.githubusercontent.com/15086658/163260697-5de63e4f-daf4-4853-81ff-5d97abc3a6f1.png)
I think we need to fix this inconvenience before merging this PR.